sdk/search/README.md

```diff
@@ -15,14 +15,15 @@ persisted index that you create, own, and store in your service.
 
 Functionality is exposed through several client libraries:
 
-- [Azure.Search.Documents](./Azure.Search.Documents/) is a preview experience
-  built on top of [Azure.Core](../core/Azure.Core/README.md) and the
+- [Azure.Search.Documents](./Azure.Search.Documents/) is the latest .NET client
+  library for building applications with Azure Cognitive Search.  It is built on
+  top of [Azure.Core](../core/Azure.Core/README.md) and the
```

Comment:
This should be a fully-qualified path to Azure.Core's README.md (master branch), given the new recommendations.